python/spacewalk/satellite_tools/reposync.py

```diff
@@ -1458,29 +1457,6 @@ def show_packages(self, plug, source_id):
 
             log(0, "    " + pack_status + pack_full_name + pack_size + pack_hash_info)
 
-    def _normalize_orphan_vendor_packages(self):
-        # Sometimes reposync disassociates vendor packages (org_id = 0) from
-        # channels.
-        # These orphans are then hard to work with in spacewalk (nobody has
-        # permissions to view/delete them). We workaround this issue by
-        # assigning such packages to the default organization, so that they can
-        # be deleted using the existing orphan-deleting procedure.
```

Comment:
Feels a shame that we're losing the context, maybe you can add this to the `changeOrgForOrphanVendorPackages` method?